src/output-json-alert.c

```diff
@@ -836,6 +836,8 @@ static TmEcode JsonAlertLogThreadDeinit(ThreadVars *t, void *data)
     MemBufferFree(aft->json_buffer);
     MemBufferFree(aft->payload_buffer);
 
+    LogFileDeInitCtx(aft->json_output_ctx->file_ctx, t->id);
```

Comment:
Do we do this in each logger to perform cleanup at shutdown we should have always been doing?  Its just clearer when doing in plugin context?

Reply:
Hmm.. Nevermind. We have to do it in all possible loggers as its the only place to run some generic cleanup right? Thats unfortunate, but shouldn't block us on this. Just points at a pain point in the logging code I think.

Reply:
This call is necessary .. for the reasons stated.